Sources/Testing/Running/Runner.RuntimeState.swift

```diff
@@ -48,16 +48,6 @@ extension Runner {
     }
 
     configuration.eventHandler = { [oldEventHandler = configuration.eventHandler] event, context in
-#if !SWT_NO_FILE_IO
```

Comment:
Do you think it would make sense to further expand on this method's doc comment to help avoid this sort of problem happening again in the future?